mitosheet/src/utils/actions.tsx

```diff
@@ -47,7 +47,7 @@ export const createActions = (
     const actions: Record<ActionEnum, Action> = {
         [ActionEnum.Add_Column]: {
             type: ActionEnum.Add_Column,
-            shortTitle: 'add col',
+            shortTitle: 'Add Col',
```

Comment:
I have some vague memory that there was a reason this needed to be lower case. Let me see if I added documentation anywhere

Reply:
ooo well I see that the type previously said it should be lowercase, but I didn't say why. I did a bit of testing and this seems to work!